src/applications/disability-benefits/all-claims/subtask/pages/appeals.jsx

```diff
@@ -77,7 +77,6 @@ const AppealsPage = ({ data = {}, error, setPageData }) => {
             value={value}
             label={label}
             checked={value === data[dataKey]}
-            uswds={false}
```

Comment:
Looks like this pr is reverting changes from #27683 and #27724. Is that intended?

Reply:
Hi @christinec-fftc this change is not intended. I'm working on cleaning up this branch and adding these changes back shortly